Update/some_004.txt

```diff
@@ -2720,7 +2720,7 @@ void main()
 	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#c8647c>暁</color>", NULL, "<color=#c8647c>Akira</color>", NULL, Line_ContinueAfterTyping); }
 	ModPlayVoiceLS(4, 39, "ps3/s04/39/310200017", 256, TRUE);
 	OutputLine(NULL, "「……うん」",
-			NULL, "\"......Ok.\"", Line_Normal);
+			NULL, "\"......Okay.\"", Line_Normal);
```

Comment:
Another far-reaching change, but one worth looking into. The console arc translations tend to use the incorrect lowercase "ok" instead of "OK" or "okay." I've left it alone so far, since it's a bit of an editing style thing, but would recommend "okay" as Meera suggests above. **Note: This would have to be changed in all console arc scripts.**

Reply:
@Darbury do you remember how this is in main arcs?

Reply:
Chapters 1-8 use "okay" throughout.